lua/lean/lsp.lua

```diff
@@ -16,7 +20,7 @@ function M.enable(opts)
   opts.handlers = {
     ["$/lean/plainGoal"] = M.handlers.plain_goal_handler;
   }
-  require('lspconfig').leanls.setup(opts)
+  require('lspconfig').lean4ls.setup(opts)
```

Comment:
I'm completely unfamiliar with the lspconfig project setup.  Is there a reason why the server definition is in nvim-lspconfig?  IMHO it would make more sense to have it here.

Reply:
That enables use of the language server without this plugin. We could include the entire default config here, but that seems redundant.

Reply:
Ok, sounds reasonable enough but I'm still not sure why anybody would want LSP integration without syntax highlighting.